packages/session-replay-browser/src/session-replay.ts

```diff
@@ -406,19 +405,23 @@ export class SessionReplay implements AmplitudeSessionReplay {
     const { apiKey } = config;
     config.apiKey = `****${apiKey.substring(apiKey.length - 4)}`;
 
+    const storageSizeData = await getStorageSize();
+
     return {
       config,
       version: VERSION,
+      ...storageSizeData,
     };
```

Comment:
```suggestion
    return {
      ...storageSizeData,
      config,
      version: VERSION,
    };
```

This is probably unlikely, but I wouldn't want `storageSizeData` to potentially override config/version